src/battle_interface.c

```diff
@@ -2438,7 +2438,11 @@ s32 MoveBattleBar(u8 battlerId, u8 healthboxSpriteId, u8 whichBar, u8 unused)
                     gBattleSpritesDataPtr->battleBars[battlerId].oldValue,
                     gBattleSpritesDataPtr->battleBars[battlerId].receivedValue,
                     &gBattleSpritesDataPtr->battleBars[battlerId].currValue,
+                #if B_FAST_HP_DRAIN == TRUE
```

Comment:
Couldn't this simply be `#if B_FAST_HP_DRAIN`?

Reply:
Other instances of boolean configs across the expansion use this format.

Reply:
> Other instances of boolean configs across the expansion use this format.

I was going to say that, but then I decided to take a deeper look to duble check, and I noticed we're actually doing preproc checks in a mixed way, just like the spacing for condition stacking 😅 
Look at this for example: https://github.com/rh-hideout/pokeemerald-expansion/blob/battle_engine/src/data/battle_anim.h#L1017-L1022